src/parser/jobs/sam/modules/ThirdEye.js

```diff
@@ -0,0 +1,86 @@
+import {ActionLink} from 'components/ui/DbLink'
+import React from 'react'
+import {Trans, i18nMark} from '@lingui/react'
+import ACTIONS from 'data/ACTIONS'
+import STATUSES from 'data/STATUSES'
+import Module from 'parser/core/Module'
+import {TieredSuggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+
+export default class ThirdEye extends Module {
+	static handle = 'Third Eye'
```

Comment:
```suggestion
	static handle = 'thirdEye'
```

Technically won't break anything, but including a space would make difficult to reference in other modules.